monkey/infection_monkey/utils/monkey_dir.py

```diff
@@ -1,16 +1,20 @@
-import os
 import shutil
 import tempfile
 
-MONKEY_DIR_NAME = "monkey_dir"
+MONKEY_DIR_PREFIX = "monkey_dir_"
+_monkey_dir = None
 
 
+# TODO: Check if we even need this. Individual plugins can just use tempfile.mkdtemp() or
```

Comment:
Isn't it still a good idea to have a distinct naming for our temporary dirs?

Reply:
Sure, but do all of the components within the agent need to store their data within the same directory? Maybe, but it could use some investigation and thought.